src/main/java/com/todoist/pojo/Model.kt

```diff
@@ -1,13 +1,13 @@
 package com.todoist.pojo
 
-open class Model(open var id: Long, open var isDeleted: Boolean) {
-    open fun hasTempId() = id < 0
+open class Model(open var id: String, open var isDeleted: Boolean) {
+    open fun hasTempId() = id.isNotEmpty() && id[0] == '-'
```

Comment:
I'd just remove this to not (anymore) make assumptions about the structure of `id`. And the API won't ever return a temp id in the model anyway.

Reply:
I just tried to keep our customs and traditions. 😅 

Fixed. ✅